spec/features/user_visits_item_show_spec.rb

```diff
@@ -44,4 +45,24 @@
     visit item_path(review.item)
     expect(page).to_not have_content("Edit or Delete")
   end
+
+  scenario 'admin can delete somebody else\' review' do
+    user1 = FactoryGirl.create(:user)
+    user2 = FactoryGirl.create(:user, role: 'admin')
+    review = FactoryGirl.create(:review, user: user1)
+
+    sign_in(user2)
+    visit item_path(review.item)
+    expect(page).to have_content("Edit or Delete")
```

Comment:
You should actually go through and actually delete the review, not just check that the button is there.